### DISCUSSION TITLE
Status not changing error is back

### DISCUSSION THREAD

**Discussion by JoshuaBarawa** [devlikeapro/waha#205] [Q&A]
Hi,
My status is not changing even after successfully scanning the qr_code. The status remains SCAN_QR_CODE even though I can get my WhatsApp screenshot. This is preventing me from sending messages.
I'm using the latest version of WAHA.
{
  "version": "2023.8.23",
  "tier": "PLUS",
  "browser": "/usr/bin/chromium"
}

This is the error I'm getting if I try to send a message
{
  "statusCode": 500,
  "timestamp": "2023-09-12T10:31:32.299Z",
  "exception": {
    "stack": "Error: Evaluation failed: TypeError: Cannot read properties of undefined (reading 'WidFactory')\n    at __puppeteer_evaluation_script__:2:42\n    at ExecutionContext._evaluateInternal (/app/node_modules/whatsapp-web.js/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:221:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async ExecutionContext.evaluate (/app/node_modules/whatsapp-web.js/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:110:16)\n    at async Client.sendMessage (/app/node_modules/whatsapp-web.js/src/Client.js:869:28)\n    at async /app/node_modules/@nestjs/core/router/router-execution-context.js:46:28\n    at async /app/node_modules/@nestjs/core/router/router-proxy.js:9:17",
    "message": "Evaluation failed: TypeError: Cannot read properties of undefined (reading 'WidFactory')\n    at __puppeteer_evaluation_script__:2:42",
    "name": "Error"
  },
  "request": {
    "path": "/api/sendText",
    "method": "POST",
    "body": {
      "chatId": "254717916656@c.us",
      "mentions": [
        "123456789@c.us"
      ],
      "text": "Hi there!",
      "session": "1234"
    },
    "query": {}
  },
  "version": {
    "version": "2023.8.23",
    "tier": "PLUS",
    "browser": "/usr/bin/chromium"
  }
}

Please help

Thanks

**Comment by allburov** [comment]
Hi! Looking at this 
https://github.com/devlikeapro/whatsapp-http-api/issues/204

**Comment by allburov** [comment]
Could you try to use the latest version `2023.9.7`?

**Reply by JoshuaBarawa** [reply]
Do I have to pay for the new credentials?

**Reply by allburov** [reply]
If you're supporters - not, you can get it on the latest post on boosty or patreon!
- https://www.patreon.com/posts/waha-2023-9-key-89093557
- https://boosty.to/wa-http-api/posts/aa7da7ea-39de-4cc8-bfbd-3676841e875d
